.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,11 @@
-<!-- Please take a look at our Contributing documentation before submitting a Pull Request!
-https://rook.io/docs/rook/latest/Contributing/development-flow/
+<!-- Thank you for contributing to Rook! -->
 
-Thank you for contributing to Rook! -->
-
-**Description of your changes:**
+<!-- STEPS TO FOLLOW:
+  1. Add a description of the changes (frequently the same as the commit description)
+  2. Enter the issue number next to "Resolves #" below (if there is no tracking issue resolved, **remove that section**)
+  3. Review our Contributing documentation at https://rook.io/docs/rook/latest/Contributing/development-flow/
+  4. Review the checklist below
```

Comment:
maybe instead of only mentioning to review the checklist, we could mention that the items can be created as   ticked at PR creation time by putting ab `x` between the brackets in the PR description editor, but alternatively, the items can be ticked interactively after PR creation by clicking the boxes in the GUI.

Reply:
My hope is that they actually do them before they are reminded by the person reviewing. If they check them off or not is up to them, and I don't see a need to require it. Then it just causes the behavior of checking them off without doing them.